package.json

```diff
@@ -51,6 +51,11 @@
   },
   "devDependencies": {
     "@types/react-dom": "^18.0.4",
-    "@types/styled-components": "^5.1.25"
+    "@types/styled-components": "^5.1.25",
```

Comment:
As we will use tailwind, you can remove styled-component stuff. Here in package.json and the ThemeProvider file in designSystem folder.